src/rebar_cmd_prv.erl

```diff
@@ -115,12 +115,15 @@ handle_cmd({error, Error}, _Opts, _State) ->
 handle_cmd({ok, {{CmdName, Info}, died}}, _Opts, State) ->
     rebar_api:error("Command ~s died unexpectedly with ~p.", [CmdName, Info]),
     {ok, State};
-handle_cmd({ok, {CmdName, Result}}, Opts, State) ->
+handle_cmd({ok, {CmdName, RawResult}}, Opts, State) ->
+    Result = string:trim(RawResult),
     case get_opt(verbose, Opts, CmdName) of
         true ->
-            rebar_api:warn("Command ~s resulted in: ~p", [CmdName, Result]);
+            rebar_api:debug("Command ~s finished", [CmdName]),
+            rebar_api:console("~ts", [Result]);
```

Comment:
Please add `console` function to the above ignore_xref definitions so that the CI job passes.
Thank you.